pkg/resourceinterpreter/customized/declarative/configurable.go

```diff
@@ -51,15 +52,15 @@ func NewConfigurableInterpreter(informer genericmanager.SingleClusterInformerMan
 }
 
 // HookEnabled tells if any hook exist for specific resource gvk and operation type.
-func (c *ConfigurableInterpreter) HookEnabled(kind schema.GroupVersionKind, operationType configv1alpha1.InterpreterOperation) bool {
-	accessor, exist := c.getCustomAccessor(kind)
-	if !exist {
-		return exist
+func (c *ConfigurableInterpreter) HookEnabled(kind schema.GroupVersionKind, operationType configv1alpha1.InterpreterOperation) (bool, error) {
+	accessor, exist, err := c.getCustomAccessor(kind)
```

Comment:
```suggestion
	accessor, enable, err := c.getCustomAccessor(kind)
```
It's not a functional issue, just to maintain consistent style.